src/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -76,7 +76,7 @@ private ReadOnlySequence(object startSegment, int startIndexAndFlags, object end
         /// Creates an instance of <see cref="ReadOnlySequence{T}"/> from linked memory list represented by start and end segments
         /// and corresponding indexes in them.
         /// </summary>
-        public ReadOnlySequence(IMemoryList<T> startSegment, int startIndex, IMemoryList<T> endSegment, int endIndex)
+        public ReadOnlySequence(ReadOnlySequenceSegment<T> startSegment, int startIndex, ReadOnlySequenceSegment<T> endSegment, int endIndex)
```

Comment:
Can you please Ctrl+F and check that we don't have `IMemoryList` mentioned in comments or documentation.

Reply:
Nowhere in corefx